local-proxy/package.json

```diff
@@ -25,6 +25,7 @@
     "bunyan": "^1.8.12",
     "bunyan-rotating-file-stream": "^1.6.3",
     "express": "^4.17.1",
+    "fs-extra": "^8.1.0",
```

Comment:
I know that @arielshaqed has strong feelings about this package. Is there another alternative?

Reply:
There's `ncp`, but I don't see a reason to change since we are already using this one.

Reply:
Makes sense